### DISCUSSION TITLE
Days Gone, Far Cry 5 and Mass Effect Andromeda

### DISCUSSION THREAD

**Discussion by tiagomullerbr** [NarutoUA/gta5_fsr#49] [Ideas]
This mod also seems to work in the Days Gone games (it sure works), Far Cry 5 and Mass Effect Andromeda. I couldn't test at higher resolutions to be sure, as I have an R7 360 and a monitor with a resolution of 1440x900. I believe it makes a bigger difference at higher resolutions. Could anyone test in any of these games?

**Comment by NarutoUA** [comment]
No it doesnt. This mod is written for gta 5 only

**Comment by tiagomullerbr** [comment]
But I noticed a difference in Days Gone. I copied the files, changed the gta5_fsr.ini to Sharpness=0.7. In the game, I set the resolution scale to 0.70. With the Mod, the game runs with a quality very similar to the native (1440x900) and with 2 fps less than without the mod. Without the mod, the game is much more blurry in this configuration and is 2 fps more. Is this possible? I didn't notice any bugs.

**Reply by NarutoUA** [reply]
Check README Troubleshooting section and post logs here from Days Gone

**Reply by tiagomullerbr** [reply]
> Verifique a seção README de solução de problemas e poste os logs aqui de Days Gone

[Days Gone.LOG](https://github.com/NarutoUA/gta5_fsr/files/6881665/Days.Gone.LOG)

**Reply by tiagomullerbr** [reply]
If you need more information, please let me know. Can I generate a screenshot of game settings. My PC configuration is an Intel Xeon E5-2630 V2 processor, 16GB RAM, 240GB Kingston SSD, 3TB Seagate HDD and XFX R7 360 2GB graphics card.

**Reply by NarutoUA** [reply]
Based on logs you provided mod doesnt work as expected

**Comment by tiagomullerbr** [comment]
In Mass Effect Andromeda and Far Cry 5, I can only see the resolution scale of each game, I didn't notice much difference like in Days Gone.
